chia/consensus/blockchain.py

```diff
@@ -144,7 +143,7 @@ async def create(
         self.constants = consensus_constants
         self.coin_store = coin_store
         self.block_store = block_store
-        self.constants_json = recurse_jsonify(self.constants)
+        self.constants_bytes = bytes(self.constants)
```

Comment:
we could even just leave it as `self.constants` here, right? and rely on built-on pickle support. It might even be faster than our streamable

Reply:
Belongs into #11758